packages/core/src/server/options.ts

```diff
@@ -7,7 +7,11 @@ export const apiOptions: APIOptions = {
   account: storeConfig.api.storeId,
   environment: storeConfig.api.environment as APIOptions['environment'],
   subDomainPrefix: storeConfig.api.subDomainPrefix ?? ['www'],
-  hideUnavailableItems: storeConfig.api.hideUnavailableItems,
+  searchOptions: {
+    hideUnavailableItems: storeConfig.api.hideUnavailableItems,
+    simulationBehavior: (storeConfig.api as Record<string, any>)
+      .simulationBehavior,
+  },
```

Comment:
question: accessing the `simulationBehavior` attribute threw a type error? Maybe we could cast the `searchOptions` instead of only a specific attribute

Reply:
Yeah, it did! Apparently because it isn't present on the faststore config default
![Screenshot 2024-09-06 at 10 44 43](https://github.com/user-attachments/assets/5510b39a-64b9-46ac-ba57-7bc38b10210e)

Could you elaborate more about your suggestion on the casting of `searchOptions`? Because the "issue" is related to the `storeConfig.api`

Reply:
Ahh got it I misunderstood the problem, that's ok then. I have no preference regarding having default value or not 🤝